packages/calendar/README.md

```diff
@@ -0,0 +1,35 @@
+## Description
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/calendar?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/calendar)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/calendar?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/calendar)
+
+```
+yarn add @spectrum-web-components/calendar
+```
+
+Import the side effectful registration of `<sp-calendar>` via:
+
+```
+import '@spectrum-web-components/calendar/sp-calendar.js';
+```
+
+When looking to leverage the `Calendar` base class as a type and/or for extension purposes, do so via:
+
+```
+import { Calendar } from '@spectrum-web-components/calendar';
+```
+
+## Example
+
+```html
+<sp-calendar></sp-calendar>
+```
+
+## To-do list
+
+-   Attribute `role="heading"` removed from `.spectrum-Calendar-title`, due to error _'The "heading" role requires the attribute "aria-level"'_
+-   Translate _"Previous"_ and _"Next"_ used in the `title` and `aria-label` of buttons displayed in the header of the calendar
+-   Translate _"Today"_ and _"selected"_ used in the `title` of the day when it is today
+-   Implement keyboard navigation
```

Comment:
Are these still "to do"?

Reply:
Yes. Most of the things involving ARIA attributes and other things related to accessibility have not been included yet, due to lack of knowledge.